packages/nodes-base/nodes/BambooHR/v1/actions/employees/get/execute.ts

```diff
@@ -0,0 +1,32 @@
+import {
+	IExecuteFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	INodeExecutionData,
+} from 'n8n-workflow';
+
+import {
+	apiRequest,
+} from '../../../transport';
+
+export async function get(this: IExecuteFunctions, index: number): Promise<INodeExecutionData[]> {
+	const body = {} as IDataObject;
+	const requestMethod = 'GET';
+
+	//meta data
+	const id = this.getNodeParameter('id', index) as string;
+
+	//query parameters
+	const fields = this.getNodeParameter('fields', index);
+
+	//endpoint
+	const endPoint = `employees/${id}/?fields=${fields}`;
```

Comment:
```suggestion
	const endPoint = `employees/${id}?fields=${fields}`;
```
I would've expected it like this.

Reply:
Updated & Fixed